src/NexusMods.App/com.nexusmods.app.metainfo.xml

```diff
@@ -29,7 +34,7 @@
     <url type="contribute">https://nexus-mods.github.io/NexusMods.App/developers/Contributing/</url>
 
     <content_rating type="oars-1.0">
-        <content_attribute id="social-info">moderate</content_attribute>
+        <content_attribute id="social-info">intense</content_attribute>
```

Comment:
Our optional, opt-in diagnostics don't (or are not meant to) share anything user identifiable; I think the previous rating of 'moderate' was actually correct.